coreneuron/nrniv/nrnoptarg.cpp

```diff
@@ -66,8 +66,8 @@ static param_int param_int_args[] = {
     {"--nwarp -W", 0, 0, 1000000, "Number of warps to balance. (0)"},
     {"--ms-subintervals", 2, 1, 2, "Number of multisend subintervals, 1 or 2. (2)"},
     {"--ms-phases", 2, 1, 2, "Number of multisend phases, 1 or 2. (2)"},
-    {"--report -r", 0, 0, 2,
-     "Enable voltage report (0 for disable, 1 for soma, 2 for full compartment)."},
+    {"--report -r", 0, 0, 3,
+     "Enable voltage report (0 for disable, 1 for soma, 2 for full compartment, 3 for synapses)."},
```

Comment:
--report option no longer needed if I am not mistaken

Reply:
true, I can remove it

Reply:
I also remove dt_report